db/offence_code_seeder_scheme_ten.rb

```diff
@@ -0,0 +1,29 @@
+  class OffenceCodeSeederSchemeTen
+    attr_reader :description, :class_letter, :contrary_to
+
+    def initialize(description, class_letter, contrary_to)
+      @description = description
+      @class_letter = class_letter
+      @contrary_to = contrary_to
+    end
+
+    def unique_code
+      modifier = 0
+      unique_code = code
+      unique_code = code(modifier += 1) while exists?(unique_code)
+      unique_code
+    end
+
+    private
+
+    def code(modifier = nil)
+      description.abbreviate +
+        modifier.to_s +
+        '_' +
+        class_letter
```

Comment:
not a blocker as can be done later but will need to be `band.to_s --> 1_1` or somesuch

Reply:
...as discussed

Reply:
It is actually passed the band
```ruby
unique_code = OffenceCodeSeederSchemeTen.new(description, offence_band.number.to_s.each_byte.to_a.join(''), row[3]).unique_code
```
but I left the signature of the method the same. Lazy copypasta :(

Renaming the parameter to `suffix` in both cases would probably be clearer

Reply:
oh right, yes, sounds good